src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$side: 100px;
```

Comment:
Consider renaming the variable $side to something more descriptive, like $cell-size or $calendar-cell-size to better reflect its purpose.